### PR TITLE
DOC: document correct_root

### LINKED ISSUE #255: DOC: stub documentation?
This doesn't seem complete
https://github.com/NSLS-II/databroker/blob/5dd77e678a4048b04594b01f88e5f4209187ca43/databroker/assets/base_registry.py#L695

### PR DESCRIPTION
**Pull request by CJ-Wright**
Closes #255

### CODE REVIEW

File: databroker/assets/base_registry.py [hunk 1]
```diff
@@ -689,15 +689,28 @@ def safe_join(inp):
                                          cmd_kwargs=dict(shift=shift),
                                          cmd='shift_root')
 
-    def correct_root(self, resource, new_root, verify):
```
Comment (tacaswell) [inline databroker/assets/base_registry.py:692]: please do not change argument names or document this as an API change.

Comment (CJ-Wright) [reply databroker/assets/base_registry.py:692]: How do you want this documented as an API change? (Sorry I don't know the workflow here)

Comment (danielballan) [reply databroker/assets/base_registry.py:692]: Why change this argument name? Does it no long accept uids?

Comment (CJ-Wright) [reply databroker/assets/base_registry.py:692]: It seems like it didn't before this PR

**Comment by CJ-Wright** [comment]
@tacaswell I think I document this properly (added info into an api doc) can I get another round of review?

File: databroker/assets/base_registry.py (hunk 1 above)
Comment (danielballan) [reply databroker/assets/base_registry.py:692]: Oh, I was reading the diff backward. Carry on.

File: doc/api-changes/correct_root_2016-08-15.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+`correct_root` supports uids
```
Comment (danielballan) [inline doc/api-changes/correct_root_2016-08-15.rst]: You have discovered an old directory full of API changes snippets that was never actually included in the rendered HTML documentation. Someone else was also (understandably) confused by this and I recently deleted `doc/api-changes` in a pull request that came after you created this branch. Use `doc/source/whats_new/v0.9.0.txt`.

Comment (danielballan) [reply doc/api-changes/correct_root_2016-08-15.rst]: (I suggest just adding one bullet point until "Internal API Changes" with all of these facts in it.

Comment (danielballan) [reply doc/api-changes/correct_root_2016-08-15.rst]: New bullet point looks good. Please remove this file.

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**